### DISCUSSION TITLE
2007/12/25/%E5%87%A0%E4%B8%AA%E5%B8%B8%E7%94%A8%E5%8D%9A%E5%AE%A2%E6%9C%8D%E5%8A%A1%E7%BD%91%E7%AB%99%E7%9A%84%E6%AF%94%E8%BE%83

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#197] [Announcements]
# 2007/12/25/%E5%87%A0%E4%B8%AA%E5%B8%B8%E7%94%A8%E5%8D%9A%E5%AE%A2%E6%9C%8D%E5%8A%A1%E7%BD%91%E7%AB%99%E7%9A%84%E6%AF%94%E8%BE%83

我最早接触博客用的是 blogpost，现在已经变成 google 网络服务的一部分了。它是最早一批的博客服务网站，并且功能强大，在国外一度占了主导地位。可惜的是这个网站在中国还没怎么流行起来就被政府和谐了。

https://blog.qizhen.xyz/2007/12/25/%E5%87%A0%E4%B8%AA%E5%B8%B8%E7%94%A8%E5%8D%9A%E5%AE%A2%E6%9C%8D%E5%8A%A1%E7%BD%91%E7%AB%99%E7%9A%84%E6%AF%94%E8%BE%83

**Comment by ruanqizhen** [comment]
之前博客下的留言：

斯
2007年12月26日 12:45 编辑
可以用sohu的，界面简单，功能强大，还可以把space的东西搬家
不过貌似速度比baidu慢一点

回复
Coral
2007年12月26日 11:29 编辑
改天我去搞个公司，聘你做CTO！

回复
Coral
2007年12月26日 11:28 编辑
Ruan Qizhen, 你在NI写程序真是浪费啦！

回复
xia
2007年12月25日 17:59 编辑
自从打开不了blogger之后…就处于极度郁闷的状态…_ _||…废了一个家…
最怕它今天封这个明天封那个的了…他娘的…苦了我们这些可怜的小网虫…

回复
